algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/models.py

```diff
@@ -264,6 +264,24 @@ def __init__(self, config: ConformerConfig):
         batch_first=True)
     self.qs = QueryScaler(dim=config.encoder_dim // config.num_attention_heads)
 
+  def _scaled_in_proj_weight(self):
+    # Scale the query projection weight.
+    in_proj_queryW_scaled = self.qs(self.in_proj_weight[:self.embed_dim].view(
```

Comment:
Nit: for better readability we could split this in two different expressions, e.g.
```python
qs_input = self.in_proj_weight[:self.embed_dim].view(
    self.num_heads, self.embed_dim // self.num_heads, -1).transpose(1, 2)
# Scale the query projection weight.
in_proj_queryW_scaled = self.qs(qs_input).transpose(1, 2).view(
    *self.in_proj_weight[:self.embed_dim].shape)
```